spk/salt-minion/src/service-setup.sh

```diff
@@ -0,0 +1,35 @@
+PYTHON_DIR="/usr/local/python3"
+SALT_PKG_DIR="/usr/local/salt-minion"
```

Comment:
Same as ${SYNOPKG_PKGDEST} in fact, so I would prefer to use that standard variable instead.

Reply:
I used a different variable because SYNOPKG_PKGDEST seemed to be setup only at install time. When I used the start-stop script after install, the variable SYNOPKG_PKGDEST was empty
I will move configuration back to ${SYNOPKG_PKGDEST}/var because as you said, I misunderstood your recommendation